### PR TITLE
Consolidated Cyrillic update v3.003

### PR DESCRIPTION
**Pull request by alexeiva**
Duplicates https://github.com/Colophon-Foundry/google-sans/pull/240

Consolidates all updates from https://github.com/Colophon-Foundry/google-sans/pull/228:

- Added new glyphs (fixes https://github.com/Colophon-Foundry/google-sans/issues/222 and https://github.com/Colophon-Foundry/google-sans/issues/212):

### New Glyphs

Roman
```
apostrophemod
tetse-cy.sc
```
Italic
```
apostrophemod
tetse-cy.sc
ge-cy.loclSRB
gje-cy.loclMKD
```

Adds final refinements:

- `Ustrait-cy` renamed to `Ustraight-cy` (misspelled name in Glyphs App lists), updated features
- Updated kerning for Cyrillic small caps. (Previously they relied on UC classes)
- in Italics added new glyph `gje-cy.loclMKD` and updated `language MKD` definition:

```
script cyrl;
language SRB;
sub be-cy by be-cy.loclSRB;
sub de-cy by de-cy.loclSRB;
sub pe-cy by pe-cy.loclSRB;
sub te-cy by te-cy.loclSRB;
sub sha-cy by sha-cy.loclSRB;
language MKD;
sub ge-cy by ge-cy.loclMKD;
sub gje-cy by gje-cy.loclMKD;
sub be-cy by be-cy.loclSRB;
sub de-cy by de-cy.loclSRB;
sub pe-cy by pe-cy.loclSRB;
sub te-cy by te-cy.loclSRB;
sub sha-cy by sha-cy.loclSRB;
```

---

@chrissimpkins edits below:

### TODO

- [ ] confirm that the SS08 feature code edit to remove Bulgarian locale definition is removed as part of this merge.  SS08 is intended to support Bulgarian in locales that do not have locale definition support.  This occurs in e.g., Sketch application.  D/W Alexei during meeting on July 17.  Implementation will be removal of the bulgarian lang registration in SS08 **only**.
- [ ] there are edited glyphs that need to be merged (*-cy suffix)
- [ ] there are many new kerning updates that need to be merged (most are *-cy glyph name or @cyr_ glyph class filtereable)

### CODE REVIEW

**Comment by chrissimpkins** [comment]
Alexei-

The decision was made to eliminate interpolation on the optical size axis.  We will be moving the opsz 14 master to 17 and keeping the 18 where it is to accomplish this.  Edd is reviewing the approach to brace/bracket layers.  

I am planning to merge your changes here early in Q3.  Can I ask you to keep an eye on the repository for these opsz axis changes and rebase your masters on the changes as they are implemented?  We'll post in the respective threads how the brace / bracket layers will be handled once this is decided.  If you have any feedback, please let us know. Thanks!

**Comment by chrissimpkins** [comment]
Tracker for opsz axis transition work https://github.com/googlefonts/googlesans/issues/49

**Comment by alexeiva** [comment]
> Alexei-
> 
> The decision was made to eliminate interpolation on the optical size axis. We will be moving the opsz 14 master to 17 and keeping the 18 where it is to accomplish this. Edd is reviewing the approach to brace/bracket layers.
> 
> I am planning to merge your changes here early in Q3. Can I ask you to keep an eye on the repository for these opsz axis changes and rebase your masters on the changes as they are implemented? We'll post in the respective threads how the brace / bracket layers will be handled once this is decided. If you have any feedback, please let us know. Thanks!

Chris, 
I've updated my files to match the opsz changes, and they build successfully. 
https://github.com/googlefonts/googlesans/commit/b166c11b41eaeb7140ec1ea37efe1c2a6d178852

**Comment by chrissimpkins** [comment]
Thank you very much Alexei!  I checked the Cyrillic and there did not appear to be any bracket or brace layers in the previously merged work. Are there any in the new designs?

**Comment by alexeiva** [comment]
> Thank you very much Alexei! I checked the Cyrillic and there did not appear to be any bracket or brace layers in the previously merged work. Are there any in the new designs?

no, there aren't any new layers in Cyrillic. I simply coordinated the changes in Latin.

**Comment by chrissimpkins** [comment]
> > Thank you very much Alexei! I checked the Cyrillic and there did not appear to be any bracket or brace layers in the previously merged work. Are there any in the new designs?
> 
> 
> 
> no, there aren't any new layers in Cyrillic. I simply coordinated the changes in Latin.
> 
> 
> 
> 

OK thanks!

**Comment by chrissimpkins** [comment]
v3.001 will be a release with the new variable font format.  The Cyrillic work here will be pushed in a release after the Hebrew work in #56 is merged.

**Comment by chrissimpkins** [comment]
TODO list in the OP was updated to remind me that we need to merge the Bulgarian locale definition changes in SS08 as part of this merge.  Alexei pointed out today that this influences the expected behavior with that fea which is to support the ability to use Bulgarian localized forms in env's where it is not possible to set langs.  This happens in e.g., Sketch application.  Alexei showed today that it is not possible to view Bulgarian locl forms with the existing feature code.

File: source/GoogleSans/GoogleSans-Italic.glyphs [hunk 1]
```diff
@@ -1301,8 +1301,8 @@ code = "lookup ccmp_Other_1 {
 	@CombiningNonTopAccents = [brevebelowcomb cedillacomb dieresisbelowcomb dotbelowcomb macronbelowcomb ogonekcomb horncomb];
 	sub [i j]' @CombiningTopAccents by [idotless jdotless];
 	sub [i j]' @CombiningNonTopAccents @CombiningTopAccents by [idotless jdotless];
-	@Markscomb = [tonos psilivaria perispomeni oxia varia dialytikaperispomeni dialytikaoxia dialytikavaria dasiaperispomeni psiliperispomeni psili koronis dasia dasiavaria psilioxia dasiaoxia];
-	@MarkscombCase = [tonos.case psilivaria.case perispomeni.case oxia.case varia.case dialytikaperispomeni.case dialytikaoxia.case dialytikavaria.case dasiaperispomeni.case psiliperispomeni.case psili.case koronis.case dasia.case dasiavaria.case psilioxia.case dasiaoxia.case];
+	@Markscomb = [tonos psili koronis dasia dasiavaria psilioxia dasiaoxia psiliperispomeni dasiaperispomeni dialytikavaria dialytikaoxia dialytikaperispomeni varia oxia perispomeni psilivaria];
```
Comment (chrissimpkins) [inline source/GoogleSans/GoogleSans-Italic.glyphs]: Thanks Alexei!

**Comment by chrissimpkins** [comment]
I believe that we have now merged all of these changes into our v3.003 release target branch.  This PR is superseded by #82.  Will close this PR

**Pull request closed without merging**